### PR TITLE
Kill ie8 specific code in Crafty

### PR DESCRIPTION
**Pull request by starwed**
This removes several chunks of code that exist only to support IE8.

grep was helpful in finding the most explicit stuff, but I'm sure there's more.

I think there was also some stuff added to the tests to work around IE8?

### CODE REVIEW

**Comment by starwed** [comment]
Oh, I think I might have accidentally included IE9 specific stuff -- I'm now a bit confused about the `nosetter` code.  I'll have a look at that later, but if someone actually knows what's going on there, would be nice to hear from you.  :)

**Comment by sbyrnes321** [comment]
Looking good, thanks!!

Yeah, in tests/core.html, you would delete the lines with "no_getter_setters". Very obvious, you can't miss it!

File: src/core.js [hunk 1]
```diff
@@ -1573,24 +1565,6 @@ function clone(obj) {
     return temp;
 }
 
-Crafty.bind("Load", function () {
-    if (!Crafty.support.setter && Crafty.support.defineProperty) {
```
Comment (starwed) [inline src/core.js:1577]: This is the line that's confusing me -- it implies that the code below should be applied when the `defineProperty` approach is used.  (That's what IE9 supports.)

But actually, the code that folows is just the fallback IE8 approach.  Maybe the conditional is just a typo?

**Comment by starwed** [comment]
Ok, pulling the trigger on this!

**Comment by potomak** [comment]
There's also an interesting fix for IE8 at https://github.com/craftyjs/Crafty/blob/develop/src/import.js#L23 where `Crafty['import']` is used instead of `Crafty.import` as `import` seems to be a reserved work in IE8.

**Comment by potomak** [comment]
Also the `storage.js` script contains IE8 specific _evil_ code, see also the comment at https://github.com/craftyjs/Crafty/blob/develop/src/storage.js#L4-L5.

In my opinion the whole Storage component should be excluded from Crafty core and added to any project only if needed (it needlessly implements HTTP communication through at low level using `XmlHttpRequest`).

**Pull request merged**